scene/gui/split_container.cpp

```diff
@@ -55,58 +55,71 @@ Control *SplitContainer::_getch(int p_idx) const {
 	return nullptr;
 }
 
-void SplitContainer::_resort() {
-	int axis = vertical ? 1 : 0;
-
+void SplitContainer::_compute_middle_sep(bool p_clamp) {
 	Control *first = _getch(0);
 	Control *second = _getch(1);
 
-	// If we have only one element
-	if (!first || !second) {
-		if (first) {
-			fit_child_in_rect(first, Rect2(Point2(), get_size()));
-		} else if (second) {
-			fit_child_in_rect(second, Rect2(Point2(), get_size()));
-		}
-		return;
-	}
-
 	// Determine expanded children
 	bool first_expanded = (vertical ? first->get_v_size_flags() : first->get_h_size_flags()) & SIZE_EXPAND;
 	bool second_expanded = (vertical ? second->get_v_size_flags() : second->get_h_size_flags()) & SIZE_EXPAND;
 
+	// Compute the minimum size
```

Comment:
```suggestion
	// Compute the minimum size.
```
If you move a comment, you have to fix it. Such are the rules :v
(same below)

Reply:
Alright alright! :smile: